tasks/oah-update.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Checking Latest version of SonarQube.

```

Comment:
ERROR! conflicting action statements: shell, warn
![image](https://user-images.githubusercontent.com/58501088/190384799-d0728e04-de90-4033-9ce6-b811c5a92592.png)

Reply:
Removed the conflicting statement warn which had no effect in functionality. Working now